Marsiple/App/New Group/AlbumsViewController.swift

```diff
@@ -50,13 +58,17 @@ private extension AlbumsViewController {
         view.backgroundColor = .white
         view.addSubview(albumsView)
         albumsView.tableView.dataSource = self
+        albumsView.tableView.delegate = self
         albumsView.tableView.register(UITableViewCell.self, forCellReuseIdentifier: "UITableViewCell")
         albumsView.snp.makeConstraints {
             $0.edges.equalTo(view.safeAreaLayoutGuide)
         }
     }
     
-    func setupNavigationBar() {
-        navigationItem.title = "Albums"
+    func populatePhotoCollectionViewControllers() {
```

Comment:
We could use a `lazy var` for this.

Reply:
How? I tried some things but non of them worked.